tools/ports/contrib/emdawn.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2025 The Emscripten Authors.  All rights reserved.
+# Emscripten is available under two separate licenses, the MIT license and the
+# University of Illinois/NCSA Open Source License.  Both these licenses can be
+# found in the LICENSE file.
+
+
+TAG = 'v20250509.171557'
+
+EXTERNAL_PORT = f'https://github.com/google/dawn/releases/download/{TAG}/emdawnwebgpu_pkg-{TAG}.zip'
+SHA512 = '4b66bf0f64b9616a6420abdad636b3ecefab892dde8f67cd941147bfddf7920f5523ff10160c9a563ef377a0f88b2dfc033527591b2d0753d531de5cbbabde59'
+PORT_FILE = 'emdawnwebgpu_pkg/emdawnwebgpu.port.py'
+
+# contrib port information (required)
+URL = 'https://dawn.googlesource.com/dawn'
+DESCRIPTION = 'Dawn is an open-source and cross-platform implementation of the WebGPU standard'
+LICENSE = 'BSD 3-Clause License'
```

Comment:
Unfortunately the license of the port is mixed. It contains files from both Dawn/webgpu-headers (BSD 3-Clause) and Emscripten (MIT / UI/NCSA dual license)

Reply:
That is fine with me. Can you tell me what I should use?

Reply:
I don't know how to properly condense that into a string. I guess it doesn't need to be that short.
`"Some files: BSD 3-Clause License. Other files: Emscripten's license (available under both MIT License and University of Illinois/NCSA Open Source License."`

Reply:
Can we just drop these 3 fields for now?

Reply:
Since they exist in the upstream port?

Reply:
They don't exist in the upstream port...

Reply:
And that allows to do `emcc --show-ports` without having to download anything and it would require a code change to make it work... I feel like that is too much for getting rid of a string...

Reply:
I have LICENSE but not URL/DESCRIPTION. And LICENSE just says "look at the license files" which it can do because they're given out together.

Reply:
And I think it is fine to include them here. Otherwise we would need to include basically the same info in some other form (comments?) anyway.